packages/core/services/oidc-provider/oidc-provider.js

```diff
@@ -0,0 +1,133 @@
+const Redis = require('ioredis');
```

Comment:
What is redis required for?

Reply:
The node-oidc-provider library needs to store data. We chose a Redis adapter since it's already in the stack, for cache and queue management.